nixos/modules/services/web-apps/rss-bridge.nix

```diff
@@ -40,27 +60,30 @@ in
 
       user = mkOption {
         type = types.str;
-        default = "nginx";
+        default = if cfg.webserver == null then "rss-bridge" else cfg.webserver;
+        defaultText = "{option}`config.services.rss-bridge.webserver` or \"rss-bridge\"";
         description = ''
           User account under which both the service and the web-application run.
         '';
       };
 
       group = mkOption {
         type = types.str;
-        default = "nginx";
+        default = if cfg.webserver == null then "rss-bridge" else cfg.webserver;
+        defaultText = "{option}`config.services.rss-bridge.webserver` or \"rss-bridge\"";
         description = ''
           Group under which the web-application run.
```

Comment:
```suggestion
          Group under which the web-application runs.
```

Reply:
thanks i even simplified the `default` expression.